lib/voxpupuli/test/facts.rb

```diff
@@ -51,7 +51,7 @@ def add_facts_for_metadata(metadata)
   metadata['dependencies'].each do |dependency|
     case normalize_module_name(dependency['name'])
     when 'camptocamp/systemd', 'puppet/systemd'
-      add_custom_fact :systemd, ->(_os, facts) { facts['service_provider'] == 'systemd' }
+      add_custom_fact :systemd, ->(_os, facts) { facts[:service_provider] == 'systemd' }
```

Comment:
Let's make it respect `RSpec.configuration.facterdb_string_keys` like on line 74.
```suggestion
      add_custom_fact :systemd, ->(_os, facts) do
        (RSpec.configuration.facterdb_string_keys ? facts['service_provider'] : facts[:service_provider]) == 'systemd'
      end
```